usecase/usecase-cordova-android-tests/samples/CrosswalkVersion/index.html

```diff
@@ -0,0 +1,75 @@
+<!--
+Copyright (c) 2014 Intel Corporation.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+* Redistributions of works must retain the original copyright notice, this list
+  of conditions and the following disclaimer.
+* Redistributions in binary form must reproduce the original copyright notice,
+  this list of conditions and the following disclaimer in the documentation
+  and/or other materials provided with the distribution.
+* Neither the name of Intel Corporation nor the names of its contributors
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Zhu, Yongyong <yongyongx.zhu@intel.com>
+
+-->
+
+<meta charset="utf-8" />
+<meta name="viewport" content="initial-scale=1.0, maximum-scale=1.0, width=device-width" />
+<link rel="stylesheet" type="text/css" href="../../css/bootstrap.css">
+<link rel="stylesheet" type="text/css" href="../../css/main.css">
+<script src="../../js/jquery-2.1.3.min.js"></script>
+<script src="../../js/bootstrap.min.js"></script>
+<script src="../../js/common.js"></script>
+<script src="../../js/tests.js"></script>
+<div id="header">
+  <h3 id="main_page_title"></h3>
+</div>
+<div class="content">
+  <h4>Note</h4>
+  <ol>
+    <li>It's only for Cordova version 4.0</li>
+  </ol>
+  <h4>Precondition</h4>
+  <ol>
+    <li>Require the latest Cordova CLI, and must >= 5.0.0
+      <div>
+        $ sudo npm install cordova -g
+      </div>
+    </li>
+  </ol>
+  <h4>Test Steps</h4>
+  <ol>
+    <li>Enter 'usecase-cordova-android-tests/res/CrosswalkVersion' directory</li>
+    <li>Run the python script:(the version is the crosswalk main version, such as 14)<br/>
+      For embedded mode: $ python test.py -m embedded -v &lt;version&gt;<br/>
+      For Shared mode: $ python test.py -m shared -v &lt;version&gt;<br/>
+    </li>
+    <li>Check the command output</li>
+    <li>Check if there are 4 apk files in 'usecase-cordova-android-tests/res/<br/>CrosswalkVersion' directory</li>
+    <li>Install and launch these apks one by one</li>
+    <li>Check if the Crosswalk version on every app page is the latest version on <br/>
```

Comment:
Should check the Crosswalk version on each app homepage are the same and should be the latest stable version of Crosswalk main version you input before.